### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by zramsay**

### CODE REVIEW

**Comment by silasdavis** [comment]
Ah great I'll review shortly. Can you `git commit --amend --signoff --no-edit` to meet the DCO check.

**Comment by zramsay** [comment]
done

File: README.md [hunk 1]
```diff
@@ -1,67 +1,69 @@
...
+
+```
+burrow spec -p1 -f1 | burrow configure -s- -v0
+```
+
+which translates into:
+
+```
+burrow spec --participant-accounts=1 --full-accounts=1 > accounts.json
+burrow configure --genesis-spec=accounts.json --validator-index=0 > burrow.toml
```
[the remark this answers is not in the document]
Comment (zramsay) [reply README.md]: lets.

File: README.md [hunk 2]
```diff
@@ -1,67 +1,69 @@
...
+burrow
+```
 
-A commented template config will be written as part of the `monax chains make` [process](https://monax.io/docs/getting-started) and can be edited prior to the `monax chains start` [process](https://monax.io/docs/getting-started).
+and the logs will start streaming through.
 
 ### Logging
-Logging is highly configurable through the `config.toml` `[logging]` section. Each log line is a list of key-value pairs that flows from the root sink through possible child sinks. Each sink can have an output, a transform, and sinks that it outputs to. Below is a more involved example of than the one appearing in the default generated config of what you can configure: 
+
+Logging is highly configurable through the `burrow.toml` `[logging]` section. Each log line is a list of key-value pairs that flows from the root sink through possible child sinks. Each sink can have an output, a transform, and sinks that it outputs to. Below is a more involved example of than the one appearing in the default generated config of what you can configure:
```
[the remark this answers is not in the document]
Comment (zramsay) [reply README.md]: i haven't played with the logging yet to know enough to write about it

File: README.md [hunk 3]
```diff
@@ -1,67 +1,69 @@
...
-Dependency management for Burrow is managed with [glide](github.com/Masterminds/glide), and you can build Burrow from source by following
 
-- [Install go](https://golang.org/doc/install)
+- [Install go](https://golang.org/doc/install) and have `$GOPATH` set
 - Ensure you have `gmp` installed (`sudo apt-get install libgmp3-dev || brew install gmp`)
-- and execute following commands in a terminal:
-- `go get github.com/Masterminds/glide`
-- `go get -d github.com/hyperledger/burrow`
-- `REPO=$($GOPATH/src/github.com/hyperledger/burrow)`
-- `cd $REPO && glide install`
```
Comment (AyushyaChitransh) [inline README.md]: I was unable to `build` without this. Is `glide install` not required in the latest build process?

Comment (zramsay) [reply README.md]: `glide` has been replaced by the official go tool for dependency management (`dep`) on the #develop branch

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (zramsay) [reply README.md]: #703

[the remark this answers is not in the document]
Comment (zramsay) [reply README.md]: #703

**Pull request merged**